scripts/common.sh

```diff
@@ -151,9 +151,9 @@ download_file_if_necessary() {
 
   if [ ! -f "${FILE_TO_DOWNLOAD_PATH}" ]; then
     curl \
-      --fail \
+      -f \
```

Comment:
MacOS curl supports `--fail` and `--output`. Why change it?

Reply:
was trying to use short options as much as possible, reverted.